.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,17 @@
+# Note: additional setup is required, see https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
+
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
@NotMyFault  can you change the branch to `main` ?

Reply:
There we go, looks like the default branch detection failed here.